### PR TITLE
Add a !tp get_review command to get the nomination text without posting it

### PR DESCRIPTION
**Pull request by Akarys42**
This can be used when information should be added to the post, or someone wants to review the user. 

Approved by our nomination overlord @wookie184.

### CODE REVIEW

**Review by wookie184 — CHANGES_REQUESTED** [review]
Just a small thing. 

Also, Discord doesn't seem to display the emojis properly in the file preview, at least for me, I get something like this:

image

The actual file when downloaded seems fine though, seems it could be a discord bug? I guess we could make it display with the `:thumbsup:` format, but I don't really mind leaving it as-is.

File: bot/exts/recruitment/talentpool/_review.py [hunk 1]
```diff
@@ -104,10 +118,7 @@ async def post_review(self, user_id: int, update_database: bool) -> None:
         )
 
         review = "\n\n".join(part for part in (opening, current_nominations, review_body, vote_request))
```
Comment (ToxicKidz) [inline bot/exts/recruitment/talentpool/_review.py:120]:
```suggestion
        review = "\n\n".join((opening, current_nominations, review_body, vote_request))
```

**Review by ToxicKidz — CHANGES_REQUESTED** [review]
Mostly looks good, one small suggestion.

File: bot/exts/recruitment/talentpool/_review.py (hunk 1 above)
Comment (Akarys42) [reply bot/exts/recruitment/talentpool/_review.py:120]: That's isn't my code :pensive: but thanks haha

**Review by wookie184 — APPROVED** [review]
Might make sense for line 273 in `_review.py`, to be changed to use `:eyes:` instead of `👀` since you've changed the other ones, but this looks good to me, thanks!

**Comment by Akarys42** [comment]
> Might make sense for line 273 in `_review.py`, to be changed to use `:eyes:` instead of `👀` since you've changed the other ones, but this looks good to me, thanks!

To be honest, if we were to do that they should be Unicode points because Git messes up encoding sometimes. But we can't use the emoji names everwhere, it will fail in some places. I don't mind switching everything though.

**Review by ToxicKidz — APPROVED** [review]
Looks good to me!

File: bot/exts/recruitment/talentpool/_review.py [hunk 2]
```diff
@@ -66,26 +66,40 @@ def schedule_review(self, user_id: int) -> None:
...
+        if not review:
+            return
+
+        guild = self.bot.get_guild(Guild.id)
+        channel = guild.get_channel(Channels.nomination_voting)
+
         log.trace(f"Posting the review of {user_id}")
+        message = (await self._bulk_send(channel, review))[-1]
+        if seen_emoji:
+            for reaction in (seen_emoji, "\U0001f44d", "\U0001f44e"):
```
Comment (Numerlor) [inline bot/exts/recruitment/talentpool/_review.py:80]: I think using `"\N{THUMBS UP SIGN}"` and `"\N{THUMBS DOWN SIGN}"` is a bit clearer for the emojis

**Review by Numerlor — CHANGES_REQUESTED** [review]
Some of the sentences sent to the channel are missing periods but that includes some of the string from he previous revisions - feel free to fix them if you think it's worth it.

L205 in the `_cog` module is still using the emoji version of a checkmark

**Review by ChrisLovering — COMMENTED** [review]
🔥

**Review by ChrisLovering — APPROVED** [review]
Looks good 👍

**Pull request merged**